CMakeLists.txt

```diff
@@ -13,6 +13,7 @@ check_ipo_supported(RESULT lto_supported OUTPUT error)
 
 if(lto_supported)
 	message(STATUS "IPO / LTO enabled")
+	set(CMAKE_POLICY_DEFAULT_CMP0069 NEW)
```

Comment:
It is, however after reading your comment and scratching my head, I realised there's a subtle difference between `cmake_policy(SET CMP0069 NEW)` and `set(CMAKE_POLICY_DEFAULT_CMP0069 NEW)`

My understanding (which could well be incorrect) is that we need the latter here because we're propagating this policy to the gtest source, and this is the way to do that - see the last point [here](https://cmake.org/cmake/help/git-stage/variable/CMAKE_POLICY_DEFAULT_CMPNNNN.html):
"Projects may set this variable before a call to [add_subdirectory()](https://cmake.org/cmake/help/git-stage/command/add_subdirectory.html#command:add_subdirectory) that adds a third-party project in order to set its policies without modifying third-party code." - which whilst we don't do explicitly, I'm assuming is what `FetchContent` does under the hood

Building it on windows 10 msvc seems to back this up. Omitting this line, using `cmake_policy` calls, setting the version in cmake_minimum_required to `3.15...3.24` all do nothing. The thing that stops the spam - that I know of - is this line.
If there's a better way, happy to change to that instead

I suppose this line could be moved lower/closer to just above the gtest call though to reduce its scope (not that its a big deal since we're using the NEW behaviour anyway)